ui/templates/ui/collection_list.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+{% block title %}OVS | Collection List{% endblock %}
+{% block pagetitle %}OVS: Collection List{% endblock %}
+
+{% block content %}
+
+<ul>
+  {% for collection in collection_list %}
+    <li><a href="{% url "collection-detail" collection.hexkey %}">{{ collection.title }}</a></li>
+  {% empty %}
+    <li><i>Please create a collection first</i></li>
+  {% endfor %}
+</ul>
+
+<hr>
+
+<h2>Create New Collection</h2>
```

Comment:
that is related to the fact that the current UI does not makes ajax requests with the right content type.
Anyway I put in place a quick fix and test